packages/mgt-chat/src/components/Chat/Chat.tsx

```diff
@@ -47,6 +72,25 @@ export const Chat = ({ chatId }: IMgtChatProps) => {
       chatClient.offStateChange(setChatState);
     };
   }, [chatClient]);
+
+  const onRenderMessage = (messageProps: MessageProps, defaultOnRender?: MessageRenderer): JSX.Element => {
+    if (isChatMessage(messageProps?.message)) {
+      const content = messageProps.message?.content;
```

Comment:
I'd suggest assigning messageProps?.message to a const in here, that way after the type guard the ? operator before accessing the content property is not needed